packages/office-addin-sso/src/configure.ts

```diff
@@ -9,11 +9,17 @@ import * as defaults from './defaults';
 import * as fs from 'fs';
 import { addSecretToCredentialStore, writeApplicationData } from './ssoDataSettings';
 import { ManifestInfo, readManifestFile } from 'office-addin-manifest';
+import * as usageDataHelper from './usagedata-helper';
+let usageDataInfo: Object = {};
 require('dotenv').config();
 
 export async function configureSSOApplication(manifestPath: string, port: string) {
+    // Log start time for configuration process
+    const startSsoCongiguration = (new Date()).getTime();
+
     // Check to see if Azure CLI is installed.  If it isn't installed, then install it
     const cliInstalled = await isAzureCliInstalled();
+
     if(!cliInstalled) {
         console.log(chalk.yellow("Azure CLI is not installed.  Installing now before proceeding"));
```

Comment:
there really shouldn't be console.log() calls and chalk used from the package. I would recommend throwing an exception and having the command top-level exception handler print the message and then have the user re-run the command after azure cli is installed.

Reply:
We shouldn't be using console.log anywhere in office-addin-scripts packages. The code should instead return exceptions and the consumer of the package can decide what UI to use for any errors. For command-line, the exception would bubble up to the top-level catch in the command which would print the exception info to the console.

Reply:
@nokafor are we really fine with removing all status logging from the package?  If so, I can remove the console logging, but I feel like this gives users a sense of progress being made